Jenkinsfile

```diff
@@ -68,7 +68,7 @@ node('JenkinsMarathonCI-Debian8') {
           try {
               timeout(time: 20, unit: 'MINUTES') {
                 withEnv(['RUN_DOCKER_INTEGRATION_TESTS=true', 'RUN_MESOS_INTEGRATION_TESTS=true']) {
-                   sh "sudo -E sbt -Dsbt.log.format=false coverage test coverageReport"
+                   sh "set -e -o pipefail; sudo -E sbt -Dsbt.log.format=false coverage test coverageReport | head -c 268435456"
```

Comment:
I think we need the `2>&1`, still. (if log output is going to STDERR then the head command will not apply)